src/tenantDataMigrations/pre-migration-integrity-check.js

```diff
@@ -0,0 +1,393 @@
+#!/usr/bin/env node
+'use strict'
+
+require('dotenv').config({ path: '../.env' })
+const { Sequelize, QueryTypes } = require('sequelize')
+
+class DatabaseIntegrityChecker {
+	constructor() {
```

Comment:
@sumanvpacewisdom keep the database connection in one file, so we can use it in any where by including that file